visualizer/flame-graph-tooltip-content.js

```diff
@@ -0,0 +1,86 @@
+'use strict'
+const d3 = require('d3')
```

Comment:
since #26 we no longer depend on the full d3, it should work if you use `./d3.js` instead